test/saithrift/tests/sail3.py

```diff
@@ -1477,3 +1477,194 @@ def runTest(self):
             self.client.sai_thrift_remove_router_interface(rif_port_id)
             self.client.sai_thrift_remove_virtual_router(self.vr_id)
             print "END OF TEST"
+@group('lag')
+@group('l3')
+class L3MultipleEcmpLagTest(sai_base_test.ThriftInterfaceDataPlane):
+    total_changing_ports = 15
```

Comment:
it's not very clear what total_changing_ports mean, can we rename to total_output_ports?